packages/web-components/src/global/colors.css

```diff
@@ -654,7 +665,7 @@
   --ic-footer-pressed: var(--ic-action-light-bg-pressed);
 
   /* HERO */
-  --ic-hero-background: var(--ic-brand-color-tertiary);
+  --ic-hero-background: var(--ic-state-layer-primary);
```

Comment:
just checking this is correct, as the hero background\that old brand-color-tertiary token should get calculated based on the brand color set. if it is now set to a fixed color, it will never update

Reply:
update: i don't think --ic-hero-background is actually used anywhere. In ic-hero it uses --ic-brand-color-tertiary directly

```
:host {
  display: block;
  background-color: var(--ic-brand-color-tertiary);
```